api/v1/cluster_types.go

```diff
@@ -905,24 +905,34 @@ type ClusterStatus struct {
 	Certificates CertificatesStatus `json:"certificates,omitempty"`
 
 	// The first recoverability point, stored as a date in RFC3339 format.
-	// This field is calculated from the content of FirstRecoverabilityPointByMethod
+	// This field is calculated from the content of FirstRecoverabilityPointByMethod.
+	//
+	// Deprecated: the field is not set for backup plugins. In-tree backup will be removed in 1.28.
```

Comment:
This is not true. In 1.28 we will remove only native Barman Cloud support. Volume snapshots will continue to be there.

Reply:
I've removed that